### PR TITLE
PT-2951: Improve warning message when deleting a single pedigree node…

### PR DESCRIPTION
**Pull request by marta-**
… results in removal of many other nodes

### CODE REVIEW

File: components/pedigree/resources/src/main/resources/pedigree/controller.js [hunk 1]
```diff
@@ -232,26 +232,16 @@ define([
...
-                editor.getOkCancelDialogue().show( '<br>All highlighted nodes will be removed. Do you want to proceed?<br><br>' +
-                                                   '<br><font style="font-size:95%">(all persons in a pedigree should be connected, so all nodes no longer<br>connected to the proband have to be removed as well)</font>',
-                                                   'Delete nodes?', removeSelected, unhighlightSelected );
-            } else {
-                editor.getOkCancelDialogue().show( '<br>All highlighted nodes will be removed. Do you want to proceed?<br><br>' +
-                        '<br><font style="font-size:95%">(all persons in a pedigree should be connected, so one of the disconnected sets of nodes has to be removed)</font>',
-                        'Delete nodes?', removeSelected, unhighlightSelected );
-            }
+            // ...and display a OK/Cancel dialogue, calling "removeSelected()" on OK and "unhighlightSelected" on Cancel
+            editor.getOkCancelDialogue().show( 'All highlighted family members (including any existing patient records) will be deleted. Are you sure you wish to proceed?',
```
Comment (allasm) [inline components/pedigree/resources/src/main/resources/pedigree/controller.js]: "...including any existing patient records..."
patient records will not be deleted, I would rephrase this

Comment (marta-) [inline components/pedigree/resources/src/main/resources/pedigree/controller.js]: Sorry, will do. I was under the impression that the text proposed on jira was approved.

Comment (marta-) [inline components/pedigree/resources/src/main/resources/pedigree/controller.js]: @allasm Modified with a message that was approved by Felicia.
